io.catenax.order_status/1.0.0/OrderStatus.ttl

```diff
@@ -0,0 +1,100 @@
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "M-Nbr-4711".
+:customerOrderId a bamm:Property;
+    bamm:preferredName "Customer Order ID"@en;
+    bamm:description "This is the order number on the customer side that can be given by e.g., the customer's enterprise resource planning system."@en;
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "C-Nbr-4711".
+:orderPositionStatuses a bamm:Property;
+    bamm:preferredName "Order Position Statuses"@en;
+    bamm:description "One Order consists of one or more Order Positions. Therefore the OrderStatus consists of one OrderPositionStatus per OrderPosition (one to one matching).\n"@en;
+    bamm:characteristic :OrderPositionStatusSortedSet.
+:OrderIdCharacteristic a bamm:Characteristic;
+    bamm:description "This is a proprietary order id, generated by either the manufacturer side or customer side."@en;
+    bamm:dataType xsd:string.
+:OrderPositionStatusSortedSet a bamm-c:SortedSet;
+    bamm:description "Each OrderPositionStatus must be unique. The OrderPositionStatus within the Set are sorted by the position id."@en;
+    bamm:dataType :OrderPositionStatus.
+:OrderPositionStatus a bamm:Entity;
+    bamm:properties (:positionId :totalAllocatedQuantityOnStock :quantityOrdered :customerSiteBpns [
+  bamm:property :allocatedStocks;
+  bamm:optional "true"^^xsd:boolean
+] :requiredDateOfCompletion ext-planned:manufacturerPartId);
+    bamm:preferredName "Status of Order Position"@en;
+    bamm:description "An Order Position is described by a quantity of one material (PartAsPlanned), which is requested by customer's site to be manufactured until a given time.\nThe Order Position Status includes the production status of the Order Position including already manufactured material for the Order Position."@en.
+:positionId a bamm:Property;
+    bamm:preferredName "Position ID"@en;
+    bamm:description "This is the business identifier for an Order Position."@en;
+    bamm:characteristic :OrderPositionIdCharacteristic;
+    bamm:exampleValue "PositionId-01".
+:totalAllocatedQuantityOnStock a bamm:Property;
+    bamm:preferredName "Total Allocated Quantity on Stock"@en;
+    bamm:description "The total Stock is the sum of all Material Quanities in AllocatedStocks related to this position (OrderPositionStatus.allocatedStocks)."@en;
+    bamm:characteristic ext-relationship:QuantityCharacteristic.
+:quantityOrdered a bamm:Property;
+    bamm:preferredName "Total Quantity Ordered"@en;
+    bamm:description "This is the quantity ordered by the customer for this position."@en;
+    bamm:characteristic ext-relationship:QuantityCharacteristic.
+:customerSiteBpns a bamm:Property;
+    bamm:preferredName "Bussiness Partner Number Site of Customer Site"@en;
+    bamm:description "To check the naming in Call-off standard"@en;
+    bamm:characteristic :BusinessPartnerNumberSiteCharacteristic;
```

Comment:
Is this the BPN? If so, there will also be a separate shared aspect, which might be used here

Reply:
Ëxactly, it'll be a BPN - or more concretely a BPNS.